packages/smart-contracts/src/contracts/EthereumSingleRequestProxy.sol

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.9;
+
+import './interfaces/EthereumFeeProxy.sol';
+
+/**
+ * @title EthereumSingleRequestProxy
+ * @notice This contract is used to send a single request to a payee with a fee sent to a third address
+ */
+contract EthereumSingleRequestProxy {
+  address public payee;
+  bytes public paymentReference;
+  address public feeAddress;
+  uint256 public feeAmount;
```

Comment:
Same question regarding the amounts

Reply:
Same as the `ERC20SingleRequestProxy` we pass in the full amount the contract gets to the `EthereumFeeProxy`